modules/installing-gitops-operator-using-cli.adoc

```diff
@@ -0,0 +1,62 @@
+// Module is included in the following assemblies:
+//
+// * installing-openshift-gitops
+
+:_content-type: PROCEDURE
+[id="installing-gitops-operator-using-cli_{context}"]
+= Installing OpenShift GitOps Operator using CLI
+
+[role="_abstract"]
+You can install {gitops-title} Operator from the OperatorHub using the CLI.
+
+.Procedure
+
+. Create a Subscription object YAML file to subscribe a namespace to the {gitops-title}, for example, `sub.yaml`:
+
+.Example Subscription
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-gitops-operator
+  namespace: openshift-operators
+spec:
+  channel: stable <1>
+  installPlanApproval: Automatic
+  name: openshift-gitops-operator <2>
+  source: redhat-operators <3>
+  sourceNamespace: openshift-marketplace <4> 
+----
+<1> Specify the channel name from where you want to subscribe the Operator.
+<2> Specify the name of the Operator to subscribe to.
+<3> Specify the name of the CatalogSource that provides the Operator.
+<4> The namespace of the CatalogSource. Use `openshift-marketplace` for the default OperatorHub CatalogSources.
+
+. Apply the `Subscription` to the cluster:
```

Comment:
Suggestion: The output is not coming good. Please check and rectify. This is coming as step 1 and all the steps are spilling over to the right pane.